src/licensedcode/data/rules/license-clue_any-osi_1.RULE

```diff
@@ -0,0 +1,7 @@
+---
+license_expression: any-osi
+is_license_clue: yes
+relevance: 100
+---
+
+OSI Approved License
```

Comment:
Do not do this. This is NOT about the any-osi license, either keep the generic one we have, or deprecated it, but surely do not report any-osi as a license for this, this is misleading.

Reply:
Agreed, updating.